### PR TITLE
Added blog post on WAIT element

### PR DESCRIPTION
**Pull request by jrbeaumont**
@snowleopard I realise you are busy, but if you get a chance, please have a read incase I have entirely missed the point of the WAIT element :)

### CODE REVIEW

File: _posts/2016-12-09-wait.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+This means that neither grants are given, these grant signals are not *high*,
+meaning that neither request has access to the resource. However, neither
+grant is *low* either. These both exist in a state which is somewhere in the
+middle. Due to this, it means that the mutex element is effectively stuck.
+
+Metastability in a mutex is resolved simply by waiting. Eventually, one of the
+signals will trend towards being high, and the other will therefore become
+low. This process is usually quite quick, however, it is not as
+instantaenous as if the requesrs arrived one after the other, and these delays
```
Comment (snowleopard) [inline _posts/2016-12-09-wait.md]: requesrs

File: _posts/2016-12-09-wait.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+While metastability is being resolved, the signal that is trending towards
+high will gradually reach it, but the resource it is aiming to access will
+have a threshold voltage, over which a signal is registered as high, and below
+which, is either 0, or registered as metastable.
+
+The WAIT element was designed in order to detect a trending-towards-high
+signal earlier, and quickly pass this onto a resource as an actually high
+signal, but how does this work?
+
```
Comment (snowleopard) [inline _posts/2016-12-09-wait.md:47]: I didn't get this paragraph. The WAIT element's purpose is to wait for an analogue and possibly hazardous signal from the environment (input `sig`), providing a clean output read-out interface `ctrl-san`.

File: _posts/2016-12-09-wait.md (hunk 1 above)
Comment (jrbeaumont) [reply _posts/2016-12-09-wait.md]: Fixed, thank you.

**Comment by jrbeaumont** [comment]
@snowleopard I have made some changes, feel free to let me know if it's still wrong haha.

I'm trying, I got very confused :S

**Comment by snowleopard** [comment]
@jrbeaumont Looks better!

**Comment by jrbeaumont** [comment]
@snowleopard: Mergable? haha

**Comment by snowleopard** [comment]
@jrbeaumont Go for it :) You can always revise it later as you keep improving your understanding.

**Comment by jrbeaumont** [comment]
This is true :)

**Pull request merged**